src/briefcase/platforms/windows/__init__.py

```diff
@@ -12,6 +12,8 @@
 
 class WindowsMixin:
     platform = "windows"
+    supported_host_os = {"Windows"}
+    supported_host_os_reason = "Building a MSI requires running on Windows."
```

Comment:
MSI is strictly a *packaging* format, and technically only one option of many (even though it's the only one supported at present). The actual failure mode will be VisualStudio or rcedit failing to run. I'd suggest:
```suggestion
    supported_host_os_reason = "Windows applications can only be built on Windows."
```

Reply:
Yeah; I should've changed this one when I ripped it out of `wix.py`.